env_store_test.go

```diff
@@ -143,15 +143,6 @@ func TestEnvStore_GetProviderSecrets(t *testing.T) {
 			addvault: false,
 			err:      fmt.Errorf("failed to create provider invalid: provider invalid is not supported"),
 		},
-		{
```

Comment:
The tests were failing on CI-runs, but were working locally, after spending more than enough time debugging it, I realised that the issue is how GH actions filesystem works since the last major patch.
Since this test is not that important, I ended up dropping it, to avoid more headache. :D